pkg/dao/repository_test.go

```diff
@@ -515,6 +515,25 @@ func (suite *RepositorySuite) TestListFilterMultipleVersions() {
 	assert.Equal(t, int64(quantity), count)
 }
 
+func (suite *RepositorySuite) TestSavePublicUrls() {
+	t := suite.T()
+	repoUrls := []string{"https://somepublicRepo.com/", "https://anotherpublicRepo.com/"}
+	err := GetRepositoryDao(suite.tx).SavePublicRepos(repoUrls)
+
+	assert.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```
Recently I have discovered the method `require.NoError(t, err)` that could be beneficial when checking the error returned; more info here: https://gitlab.com/gitlab-org/gitlab-pages/-/merge_requests/87  and here https://pkg.go.dev/github.com/stretchr/testify/require